Jenkinsfile

```
@@ -137,6 +137,15 @@ void rpm_test_post(String stage_name, String node) {
     job_status_update()
 }
 
+String sconsArgs() {
+    if (!params.CI_SCONS_ARGS) {
+        return sconsFaultsArgs()
+    }
+
+    println("Compiling DAOS with libasan")
```

It is not `libasan` specific argument.
```suggestion
    println("Compiling DAOS with custom arguments")
```